src/Materials/standardMaterial.ts

```diff
@@ -1860,18 +1860,6 @@ export class StandardMaterial extends PushMaterial {
         return result;
     }
 
-    /**
-     * Serializes this material in a JSON representation
-     * @returns the serialized material object
-     */
-    public serialize(): any {
-        const serializationObject = SerializationHelper.Serialize(this);
-
-        serializationObject.stencil = this.stencil.serialize();

```

Comment:
Are we not losing this info ???

Reply:
We are just inheriting from Material.ts now, which serializes all the same info

Reply:
perfect